ompi/mca/pml/ucx/pml_ucx.c

```diff
@@ -770,8 +762,10 @@ int mca_pml_ucx_iprobe(int src, int tag, struct ompi_communicator_t* comm,
     if (ucp_msg != NULL) {
         *matched = 1;
         mca_pml_ucx_set_recv_status_safe(mpi_status, UCS_OK, &info);
-    } else {
-        opal_progress();
+        ucx_progress_cntr = 0;
```

Comment:
reset is used to count "progress iterations" after every completed request to avoid unpredictable calls of opal_progress.

do you think it is bad idea?